Godeps/Godeps.json

```diff
@@ -1311,213 +1311,213 @@
 		},
 		{
 			"ImportPath": "github.com/google/cadvisor/accelerators",
-			"Comment": "v0.28.0",
-			"Rev": "3d2e7fcfa396ab4f193c7720b6a6dc3133b6f50a"
+			"Comment": "v0.28.0-14-g5231853",
```

Comment:
please wait for me to cut the next release, which is scheduled for today.   I prefer only vendoring tagged cadvisor versions.  Thanks

Reply:
I agree with `tagged cadvisor versions`. But it maybe too late for now to tag a new cadvisor release. :(

This is needed for adding disk io metrics for heapster. I will fix the PR and please take a look at this.